src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -633,9 +633,11 @@ def human_readable_size(size):
     Take a size in bytes, and transform it into a human readable size with up
     to two decimals of precision.
     """
-    suffixes = ['B', 'KB', 'MB', 'GB', 'TB']
+    suffixes = ['B', 'KB', 'MB', 'GB', 'TB', 'PB']
     suffix_index = 0
     while size > 1024:
+        if suffix_index == len(suffixes) - 1:
+            break
```

Comment:
```suggestion
    while size > 1024 and suffix_index < len(suffix)::
        if suffix_index == len(suffixes) - 1:
            break
```

Reply:
There is an superflous `:` in the suggested change and the if statement can be deleted, but otherwise :+1:

Reply:
Oops, sorry about that. Updated the suggested code. Thanks.

Reply:
why not put
```py
BINARY_DIVIDER = 1024
suffix = suffixes[0]
for suffix in suffixes:
    if size > BINARY_DIVIDER:
        size /= BINARY_DIVIDER
    else:
        break
return f"{size:.2f} {suffix}"
```